src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = new User();
+        user.setLogin("Tretiak");
+        user.setPassword("123456");
+        user.setAge(20);
+    }
+
+    @Test
+    void register_noExistUserLogin_Ok() {
+        if (Storage.people.size() != 0) {
+            assertEquals(user, registrationService.register(user));
+        }
+    }

```

Comment:
we shouldn't use `if` here
after registration we should check that user was added
so we can write one more `assertEquals` and check that `Storage.people` now contains our user